chat.go

```diff
@@ -65,7 +94,7 @@ type FunctionDefinition struct {
 	Name        string `json:"name"`
 	Description string `json:"description,omitempty"`
 	// Parameters is an object describing the function.
-	// You can pass json.RawMessage to describe the schema,
+	// You can pass a []byte describing the schema,
```

Comment:
Could you please remove this change? See: https://github.com/sashabaranov/go-openai/pull/441

Reply:
Ok, I will submit a new PR!

Reply:
New PR ,See #451 THANKS! @sashabaranov